### ISSUE TITLE
Feature request: when using Math language, copy the results when selecting a line or the whole block

### ISSUE THREAD

**Issue by rishid** [heyman/heynote#193]
**Have you searched existing Github issues to see if someone has already requested this feature?**
Yes

**Is your feature request related to a problem? Please describe.**
When using a math block, it is annoying that I cannot copy the answers. If I am doing some scratch math and want to paste it into Slack, this doesn't seem to copy the results.

**Describe the solution you'd like**
Copy results

**Describe alternatives you've considered**
None

**Comment by NoahAndrews** [comment]
Clicking on the result copies it

**Comment by rishid** [comment]
Oh that I didn't realize that was a shortcut. My request is more along the lines of being able to copy the actual line _and_ the results to share with someone.

**Comment by heyman** [comment]
I agree. I'd like to implement it so that the result for each line gets appended to each line when the text is copied (only the clipboard text is changed). When text is pasted into a Math block, the result gets removed from each line.

So, copying the following from a Math block:

```
a = 34
b = 124

Result:
a + b

1 EUR in SEK
```

Should result in something like this on the clipboard:
```
a = 34       # = 34
b = 124      # = 124

Result:
a + b        # = 158

1 EUR in SEK # = 11.255813 SEK
```

And pasting that text in a Math block should automatically remove the `# = ...` from the end of the lines.

**Comment by porridgewithraisins** [comment]
would love this as wel

**Comment by simonratner** [comment]
Tangentially, would love a keyboard shortcut for copying just the result. Often i find myself doing a calculation and then needing to paste the result into code/document/etc. I love the fact that Ctrl-C in calculator copies the result - perhaps a modified copy command could do the same, e.g.:
```
Ctrl/Cmd-C         copy line (including result as described above)
Ctrl/Cmd-Shift-C   copy result (in math block only)
Ctrl/Cmd-V         paste line (strip result if in math block as describe above)
Ctrl/Cmd-Shift-V   paste result only (if copied from math block - otherwise just regular paste)
```
This is reminiscent of "paste as plain text" feature in e.g. Google Docs (Ctrl-Shift-V).
